install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml

```
@@ -845,7 +845,7 @@ spec:
               - key: ca.crt
                 path: common-etcd-client-ca.crt
           {{- else }}
-          - configMap:
+          - {{ .Values.tls.caBundle.useSecret | ternary "secret" "configMap" }}::
```

I think there is an extra column here
```suggestion
          - {{ .Values.tls.caBundle.useSecret | ternary "secret" "configMap" }}:
```

---

Derp, thanks for catching it!